components/sections/Tabs/Tabs.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../../styles/fonts.scss";
+
+.tabs {
+  min-height: 300px;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  font-family: $poppins-semibold;
+  background-color: #31683d;
+}
+
+.tabsName {
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 30px;
+  justify-content: center;
+  align-content: center;
+  @media (max-width: 500px) {
+    flex-direction: column;
+  }
+}
+
+.tabTitle {
+  color: #ffffff;
+}
+
+.tabName {
+  font-size: 20px;
+  cursor: pointer;
+  margin-right: 30px;
+  color: #81a88f;
+  margin-left: 30px;
+  padding-bottom: 5px;
+}
+
+.tabsContent {
+  padding-left: 15vw;
+  padding-right: 15vw;
+  @media (max-width: 500px) {
+    padding-left: 10vw;
+    padding-right: 10vw;
+  }
+}
+
+.selectedTabName {
+  border-bottom-style: solid;
```

Comment:
you can replace all this with`border-bottom: 3px solid #df8832`

Reply:
You're right, I kept adding styles and I didn't realize that I had the whole set.